src/mono/mono/utils/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ if(HOST_WIN32 AND HOST_AMD64)
     set(CMAKE_ASM_MASM_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY_MultiThreadedDebugDLL "")
 
     list(APPEND utils_win32_sources win64.asm)
-elseif(HOST_WASI)
+elseif(HOST_WASI OR HOST_WASM)
```

Comment:
`HOST_WASM` should be enough for both HOST_WASI and HOST_BROWSER

Reply:
ah right 😄 fixed